### DISCUSSION TITLE
As a Project Manager, I'd like to be able to set column level access in Github Projects (for columns like estimated hours).

### DISCUSSION THREAD

**Discussion by albertlinao** [community/community#38384] [Projects and Issues]
I believe a column level access / privilege type of config would be valuable to some teams. 

For columns like hours estimate, it allows a flow where only the project owner can set hour estimates.

I'm aware that I can make individual developers have read-only-access-project-wide, but they will no longer be able to make changes to columns they're expected to be able to change.

**Comment by ossanna16** [comment]
Hi there @alberto-linao and welcome to our community! Thank you for the feedback 🙂
